src/platform/user/tests/authentication/utilities.unit.spec.js

```diff
@@ -43,16 +43,31 @@ describe('authentication URL helpers', () => {
     expect(global.window.location).to.include('/sessions/signup/new');
   });
 
+  it('should redirect for signup', () => {
```

Comment:
```suggestion
  it('should redirect for signup v1', () => {
```

Reply:
thanks for catching that, just fixed here https://github.com/department-of-veterans-affairs/vets-website/pull/12022/commits/a5f30e2e5ddd02f1ab0c451d811ba9c43853c3b3